screens/Home/ViewVcModal.tsx

```diff
@@ -57,9 +58,10 @@ export const ViewVcModal: React.FC<ViewVcModalProps> = props => {
   return (
     <Modal
       isVisible={props.isVisible}
-      onDismiss={props.onDismiss}
+      arrowLeft={<Icon name={''} />}
```

Comment:
No need to pass Empty Icon Component, instead pass boolean value. check with @srikanth716 for the latest modification on the Modal Component with conditonal rendering.

Reply:
@vijay151096, Just now discussed with @swatigoel @srikanth716 in desk check regarding to this. I will raise PR with resp. change once Srikanth's has been merged.